gallery/src/RawHtmlExample.tsx

```diff
@@ -11,6 +11,6 @@ interface Props {
 }
 
 const RawHtmlExample = ({ html }: Props) =>
-  <div className="gallery-raw-html-example" dangerouslySetInnerHTML={{ __html: html }} />;
+  <div className="gallery-example-live gallery-example-live--raw-html" dangerouslySetInnerHTML={{ __html: html }} />;
```

Comment:
This and similar changes were cherry-picked in from the checkered-background improvement day branch, in order to get the `.gallery-example-live` class on _all_ live examples (not just HTML examples).  This helps with the visual test selectors.